notebooks/04-Creating_ASDF_Files.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Maybe mention that updates may be expensive if they require rewriting the whole file (refer to padding and other mitigation strategies elsewhere, or that these can be avoided).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>